src/nets/e2e_asr_attctc_th.py

```diff
@@ -1695,6 +1695,13 @@ def recognize(self, h, recog_args):
         for l in six.moves.range(1, self.dlayers):
             c_list.append(self.zero_state(h.unsqueeze(0)))
             z_list.append(self.zero_state(h.unsqueeze(0)))
+        if rnnlm:
+            state = {
```

Comment:
It's better to make this as a method in the rnnlm.
Then, if we change the number of layers, etc., we don't have to change this part.
(chainer part as well....)

Reply:
I agree. RNNLM.forward should support `state=None` for the initial step.